.github/workflows/ci-owasp-dep-check.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
           docker rmi $(docker images -q) -f
           df -h
 
-      # Projects dependent on flume, hdfs, hbase, and presto currently excluded from the scan.
+      # Projects dependent on flume, hdfs, hbase, and trino currently excluded from the scan.
```

Comment:
This patch doesn't actually touch dependencies. OWASP failed due to this comment edition triggering validation. The failures can be resolved in another turn.